lib/src/executables/intl_message_migration.dart

```diff
@@ -199,6 +199,18 @@ void main(List<String> args) async {
     await migratePackage(
         package, packageNameLookup, processedPackages, codemodArgs, dartPaths);
   }
+
+  if (exitCode != 0) {
```

Comment:
The impression I had from the discussion was that it would be nice if it could actually list the problem code locations/strings. Some people have expressed the desire to do the changes manually using the IDE extensions rather than running the codemod.

Reply:
I share that impression from one of our conversations, yet in another conversation, there was this question: "
How do I find the change that needs to be made in this without reading 7000 lines?"  If we append to the existing logs in the CI check, we're adding to that large quantity of lines, where as if we append the 3 lines at the end and encourage folks to use the codemod locally, they will be prompted with each change, and can step through the changes needed.

I think for now, we should take an incremental step forward and see if this will meet most people's needs, then circle back with folks with whether we need to do more.  Additionally, I'd like to talk with FEA about reducing the amount of low-value codemod-related logs firts, before we add much more.

Thoughts?